### ISSUE TITLE
SPLASH not indexed by google ?

### ISSUE THREAD

**Issue by sneumann** [EBI-Metabolights/MetaboLights#8]
Hi, 
the dynamically generated pages in MetaboLights are currently 
not fully picked up by google et al. This is particularly sad 
for the SPLASH. The search https://www.google.de/search?q=MetaboLights+splash10-0002-0900000000-b112e4e059e1ecf98c5f 
should ideally return http://www.ebi.ac.uk/metabolights/MTBLC27732#ms

Indeed, JavaScript generated dynamic HTML pages are difficult for google et al. A few SPLASH people also had to work around that. Michael Wilson might be able to hint what he had to do for HMDB, Gert created a set of static files at http://mona.fiehnlab.ucdavis.edu/repository/
which are indexed by google and Tobias created a sitemap.xml for massbank.eu 
at http://massbank.eu/MassBank/sitemap.xml that makes sure google follows some of the generated
links.

HTH, yours, Steffen

**Comment by khaug** [comment]
Yes, we chose to just quickly include the splash as JS only.  We need to persist this somehow.  Thank you for the suggestions

**Comment by CS76** [comment]
The new fix that went live yesterday have all the SPLASH values pre-calculated and persisted in our server. We just need to give some time for the google to index our pages :)
